effects.cwt

```diff
@@ -2733,7 +2740,12 @@ alias[effect:set_allow_title_revokation] = bool
 # TODO: Figure out
 ## scope = title
 ### Crown laws
-alias[effect:set_allow_vice_royalties] = bool/tier
+alias[effect:set_allow_vice_royalties] = bool
+
+# TODO: Figure out
+## scope = title
+### Crown laws
+alias[effect:set_allow_vice_royalties] = tier
```

Comment:
`enum[title_tiers]`

Reply:
Done.

Reply:
Nvm it was another line.

Reply:
Strange, when I go to edit the file the line seems to be fine but here it's not.

Reply:
Fixed now according to the files changed list.